src/routes/messages/mappers/message-mapper.spec.ts

```diff
@@ -0,0 +1,210 @@
+import { faker } from '@faker-js/faker';
+import { range } from 'lodash';
+import { DataSourceError } from '../../../domain/errors/data-source.error';
+import { messageConfirmationBuilder } from '../../../domain/messages/entities/__tests__/message-confirmation.builder';
+import { messageBuilder } from '../../../domain/messages/entities/__tests__/message.builder';
+import { safeAppBuilder } from '../../../domain/safe-apps/entities/__tests__/safe-app.builder';
+import { SafeAppsRepository } from '../../../domain/safe-apps/safe-apps.repository';
+import { safeBuilder } from '../../../domain/safe/entities/__tests__/safe.builder';
+import { AddressInfoHelper } from '../../common/address-info/address-info.helper';
+import { AddressInfo } from '../../common/entities/address-info.entity';
+import { MessageConfirmation } from '../entities/message-confirmation.entity';
+import { Message, MessageStatus } from '../entities/message.entity';
+import { MessageMapper } from './message-mapper';
+
+describe('Message mapper (Unit)', () => {
```

Comment:
In my opinion most of these cases should be covered by the controller test right?

One disadvantage of this test (in comparison to the controller) is that it relies more heavily on the underlying implementation.

This type of mapping, in my opinion, is the core part of the route itself. What do you think about merged the scenarios described here with the controller tests?

Reply:
Yes, it's a good idea, I'll move these tests to the controller tests 👍

Reply:
I've moved the tests cases which were not previously covered to the controller in [1ddeeae](https://github.com/5afe/safe-client-gateway-nest/pull/275/commits/1ddeeaecf40f2b46d16314890d294c129580cd8f)